### ISSUE TITLE
dead link for firefox extention

### ISSUE THREAD

**Issue by Akiyamka** [hediet/browser-ext-github-monaco#37]
https://addons.mozilla.org/en-US/firefox/addon/monaco-markdown-github-editor/
> Oops! We can’t find that page

**Comment by hediet** [comment]
Yep, they disabled the firefox extension yesterday because I'm using `innerHTML` to set the html preview of the markdown - even though I simply do the same as the GitHub page itself.
They suggested to use https://github.com/cure53/DOMPurify to sanitize the HTML before setting it to innerHTML.

**Comment by Akiyamka** [comment]
@hediet, I hope it help
